### DISCUSSION TITLE
mc服务器招新

### DISCUSSION THREAD

**Discussion by zhenxun321** [share121/inter-knot#4704] [委托]
版本为1.21fabric版，装了间隙泉，基岩也能进
服务器内无大型MOD，but腐竹装了些奇奇怪怪的）
希望大家能友好相处（服务器一样）
总体偏休闲，适合养老，if你想要挑战也不是不行）
![qrcode_1725892858176](https://github.com/user-attachments/assets/bc9eba85-2745-4a1f-9de4-2f4e678647a4)

**Comment by wuyue200859** [comment]
在这还能见到Minecraft
